sdk/core/Azure.Core/tests/LightweightPkcs8DecoderTests.cs

```diff
@@ -186,13 +186,23 @@ public void VerifyDecoder()
             byte[] data = Convert.FromBase64String(PrivateKey);
 
             using RSA fromPem = LightweightPkcs8Decoder.DecodeRSAPkcs8(data);
-            using RSA fromPfx = new X509Certificate2(Convert.FromBase64String(Pfx)).GetRSAPrivateKey();
+
+            byte[] cer = Convert.FromBase64String(Pfx);
+            X509Certificate2 clientCert;
+
+#if NET9_0_OR_GREATER
```

Comment:
[SYSLIB0057: X509Certificate2 and X509Certificate constructors for binary and file content are obsolete](https://learn.microsoft.com/en-us/dotnet/fundamentals/syslib-diagnostics/syslib0057)